File: Python/uri_1066.py
v = []
w = 0
x = 0
y = 0
z = 0
for i in range(0, 5):
    v.append(int(input()))
    if v[i] % 2 == 0:
        w += 1
    if v[i] % 2 != 0:
        x += 1
    if v[i] > 0:
        y += 1
    if v[i] < 0:
        z += 1
print('{} valor(es) par(es)'.format(w))
print('{} valor(es) impar(es)'.format(x))
print('{} valor(es) positivo(s)'.format(y))
print('{} valor(es) negativo(s)'.format(z))

File: Go/uri_1066.go
package main

import "fmt"

func main() {
	var v [5]int
	var p, in, po, ne = 0, 0, 0, 0
	for i := 0; i < len(v); i++ {
		fmt.Scanln(&v[i])
		if v[i]%2 == 0 {
			p++
		} else {
			in++
		}
		if v[i] < 0 {
			ne++
		} else if v[i] == 0 {
			continue
		} else {
			po++
		}

	}
	fmt.Printf("%d valor(es) par(es)\n%d valor(es) impar(es)\n%d valor(es) positivo(s)\n%d valor(es) negativo(s)\n", p, in, po, ne)
}
